src/Auxillary/json_schema/include/make_schema.hpp

```diff
@@ -28,6 +28,23 @@ namespace Aux::schema {
    */
   json make_boundary_schema(int num_values);
 
+  /**
+   * @brief Helper to create an initial value configuration json schema for
+   * a component (this is then assembled into a larger initial value json)
+   *
+   * @param num_interpol_points The number of interpolation points (minimum &
+   * maximum) if this number is <=0, this is interpreted as no restriction
```

Comment:
How exactly does a number <=0 work out here?
is this just a restriction of json schema?